examples/i3s/deck.gl/i3s-3d-layer/get-frame-state.js

```diff
@@ -0,0 +1,98 @@
+import {Vector3} from 'math.gl';
+import {CullingVolume, Plane} from '@math.gl/culling';
```

Comment:
Q: Is this different from the `Tile3DLayer`? If so can we do any processing inside the loaders so they don't have to be different?

Reply:
the example folder is temporary. Will merge `i3s-3d-layer` with `tile-3d-layer` in follow-up PR

Reply:
this is a temp change, to test i3s loading and rendering, will be removed and merged with 3d-tiles in followups.